pkg/binary/envoy/controlplane/istio.go

```diff
@@ -124,8 +124,12 @@ func retrieveIPs() ([]string, error) {
 }
 
 func istioNode(cfg *envoy.Config) string {
+	mode := cfg.Mode
+	if mode == envoy.LoadBalancer {
+		mode = "router"
```

Comment:
Q: is this “router” correct?

Reply:
Yes, Istio calls it router. We want to explicitly **not** use that terminology to avoid confusion.

Reply:
👌👌👌